windows/install_vcpkg.ps1

```diff
@@ -1,6 +1,6 @@
 $ErrorActionPreference = "Stop"
 
-git clone --depth 1 --branch 2021.05.12 https://github.com/microsoft/vcpkg
+git clone --branch 2021.05.12 https://github.com/microsoft/vcpkg
```

Comment:
```suggestion
# Do not use '--depth 1' since vcpkg needs to browse its git history for dependency retrieval
git clone --branch 2021.05.12 https://github.com/microsoft/vcpkg
```